src/olympia/devhub/templates/devhub/includes/macros.html

```diff
@@ -10,6 +10,16 @@
 </p>
 {% endmacro %}
 
+{% macro markdown_tip(title=None) %}
```

Comment:
I like what you have done elsewhere migrating from "html" to "syntax" let's do that here also "supported_syntax_tip" is a more generic name that won't again need to be migrated and encapsulates what you mean about markdown without coupling the name to the language.